src/swap/SwapReviewScreen.tsx

```diff
@@ -0,0 +1,374 @@
+              <View style={styles.tokenDisplayView}>
+                <TokenDisplay
+                  style={styles.transactionDetailsRightText}
+                  amount={divideByWei(
+                    swapInfo?.unvalidatedSwapTransaction?.gas *
+                      swapInfo?.unvalidatedSwapTransaction.gasPrice
+                  )}
+                  tokenAddress={fromToken}
+                  showLocalAmount={false}
+                  testID={'EstimatedGas'}
+                />
+              </View>
+            </View>
+            <View style={styles.row}>
+              <TouchableOpacity
+                style={styles.touchableRow}
+                onPress={() => setSwapFeeModalVisible(true)}
+                hitSlop={variables.iconHitslop}
+              >
+                <Text style={{ marginRight: 4, ...fontStyles.regular }}>
+                  {t('swapReviewScreen.swapFee')}
+                </Text>
+                <InfoIcon size={12} color={colors.gray4} />
+              </TouchableOpacity>
+              <TokenDisplay
+                style={[styles.transactionDetailsRightText, !swapFeeEnabled && styles.feeWaived]}
+                amount={divideByWei(swapInfo?.unvalidatedSwapTransaction?.sellAmount).multipliedBy(
+                  swapFeeDecimal
+                )}
+                tokenAddress={fromToken}
+                showLocalAmount={true}
+                testID={'SwapFee'}
+              />
+            </View>
+          </View>
+        </ScrollView>
+      )}
+      <Button
+        style={{ padding: Spacing.Regular16 }}
+        onPress={() => Logger.debug('TODO Perform Swap!!!')}
+        text={t('swapReviewScreen.complete')}
+        size={BtnSizes.FULL}
+        disabled={loading || fetchSwapQuoteError}
+      />
+      {/** Estimated amount dialog */}
+      <Dialog
+        title={t('swapReviewScreen.estimatedAmountTitle')}
+        isVisible={estimatedModalVisible}
+        actionText={t('dismiss')}
+        actionPress={() => setEstimatedDialogVisible(false)}
+        isActionHighlighted={false}
+        onBackgroundPress={() => setEstimatedDialogVisible(false)}
+      >
+        {t('swapReviewScreen.estimatedAmountBody', {
+          slippagePercent: maxSlippagePercent,
+        })}
+      </Dialog>
+      {/** App fee dialog */}
+      <Dialog
+        title={t('swapReviewScreen.swapFeeTitle')}
+        isVisible={swapFeeModalVisible}
+        actionText={t('dismiss')}
+        actionPress={() => setSwapFeeModalVisible(false)}
+        isActionHighlighted={false}
+        onBackgroundPress={() => setSwapFeeModalVisible(false)}
+      >
+        {t('swapReviewScreen.swapFeeBody', {
+          swapFee: swapFeePercentage,
+        })}
+      </Dialog>
+    </SafeAreaView>
+  )
+}
+
+const styles = StyleSheet.create({
+  safeAreaContainer: {
+    flex: 1,
+  },
+  loadingContentContainer: {
+    justifyContent: 'flex-start',
```

Comment:
i think this is the default and not needed?

Reply:
Removed in https://github.com/valora-inc/wallet/pull/2767/commits/7d463246666579c3467bbcc2e82dff8142660a72!